man/leapp.1

```diff
@@ -1,59 +1,165 @@
+Overrides the automatically detected GRUB core location.
 .RE
 
-LEAPP_DEBUG
+.B LEAPP_NO_RHSM
 .RS 4
-This environment variable enables debug logging, see --debug for more information
+Skip actions that use Red Hat Subscription Manager. Equivalent to \fB--no-rhsm\fR, which takes precedence.
 .RE
 
-LEAPP_VERBOSE
+.B LEAPP_OVL_SIZE
 .RS 4
-This environment variable enables verbose logging, see --verbose for more information
+For each XFS partition created with \fIftype=0\fR, leapp creates an overlay file in order to proceed. This option sets the size (in MB) of every such file. Defaults to \fB2048\fR.
 .RE
 
-.SH "RETURN CODES"
 
-0 - No error occurred.
+.SS Developer variables
+These variables shouldn't be needed under normal circumstances and aren't meant for normal use. 
+
+.B LEAPP_UNSUPPORTED
+.RS 4
+Necessary in case you use any variable starting with \fBLEAPP_DEVEL\fR (see below) or use \fB--whitelist-experimental\fR.
+
+By setting this variable to 1, you essentially acknowledge that some options you might use are not fully supported and not meant for ordinary users.
+.RE 
+
+.B LEAPP_DEVEL_RPMS_ALL_SIGNED
+.RS 4
+If set, leapp will consider all installed RPMs to be signed by Red Hat and upgrade them. By default, leapp only handles RPMs signed by Red Hat. What happens with RPMs not signed by Red Hat is undefined.
+.RE
 
-1 - Any actor in a workflow reported an error (through report_error function or StopActorExecutionError exception).
+.B LEAPP_DEVEL_TARGET_RELEASE \fR<\fIversion\fR>
+.RS 4
+Changes the default target RHEL 8 minor version.
+.RE
+
+.B LEAPP_DEVEL_SKIP_CHECK_OS_RELEASE
+.RS 4
+If set, leapp will not check whether the source RHEL 7 version is the supported one.
+.RE
+
+.B LEAPP_DEVEL_DM_DISABLE_UDEV
+.RS 4
+Disables udev support in libdevmapper, dmsetup and LVM2 tools globally without a need to modify any existing configuration settings. Useful if the system environment does not use udev.
+.RE
+
+.B LEAPP_DEVEL_SOURCE_PRODUCT_TYPE \fR<\fIproduct_type\fR>
+.RS 4
+Specifies source product type. Expected values: \fBga\fR, \fBbeta\fR, \fBhtb\fR. Defaults to \fBga\fR.
+.RE
+
+.B LEAPP_DEVEL_TARGET_PRODUCT_TYPE \fR<\fIproduct_type\fR>
+.RS 4
+Specifies target product type. Expected values: \fBga\fR, \fBbeta\fR, \fBhtb\fR. Defaults to \fBga\fR.
+.RE
+
+.SH "EXIT CODES"
+.B 0
+\- No error occurred.
+
+.B 1
+\- Any actor in a workflow reported an error (through calling \fBreport_error\fR or raising \fBStopActorExecutionError\fR).
 
 
 .SH "REPORTING BUGS"
-Report bugs to the issue tracker <https://github.com/oamg/leapp-repository/issues> or to bugzilla <https://bugzilla.redhat.com> under the `Red Hat Enterprise Linux 7` product and the `leapp-repository` component.
+Report bugs to the issue tracker (\fIhttps://github.com/oamg/leapp-repository/issues\fR) or to bugzilla (\fIhttps://bugzilla.redhat.com\fR) under the `Red Hat Enterprise Linux 7` product and the `leapp-repository` component.
```

Comment:
```suggestion
Report bugs to Red Hat Bugzilla (\fIhttps://bugzilla.redhat.com\fR) under the `Red Hat Enterprise Linux 7` product and the `leapp-repository` component.
```
If I'm not mistaken, we want customers to use bugzilla only, not github issues. @shaded-enmity, right?

The same applies to the snactor man page.